src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -406,6 +406,17 @@ public static class Importer
          public static readonly string ApplyConfiguration = "Apply configuration";
          public static readonly string GroupByTitle = "Group by";
          public static readonly string MappingTitle = "Mapping";
+         public static readonly string ReloadWillCauseChangeOfDataSets = "Reloading will cause the following changes in observed data. Do you really want to reload?";
+         public static readonly string DataSetsWillBeOverwritten = "Datasets that will be overwritten";
+         public static readonly string NewDataStetsWillBeImported = "New datasets that will be imported";
+         public static readonly string ReloadData = "Reload Data";
+
+         public static string FileFormatNotSupported(string fileName)
+         {
+            return "The file: \n \n" + fileName +
```

Comment:
we should use string interpolation here... $ {} or  string builder. Using + is just allocating strings for nothing all the time. Not a big deal, but we have been diligent everywhere else"

see
ShouldWatermarkBeUsedForChartExportToClipboard for example of string.builder

Reply:
done